samples/python/benchmark/bert_benchmark/bert_benchmark.py

```diff
@@ -57,7 +57,7 @@ def main():
     sst2_sentences = sst2['validation']['sentence']
     # Warm up
     encoded_warm_up = dict(tokenizer('Warm up sentence is here.', return_tensors='np'))
-    for _ in ireqs:
+    for _ in range(len(ireqs)):
```

Comment:
Why is `range` preferred?

Reply:
It's a new rule in `flake8-bugbear` which is part of our linters https://github.com/PyCQA/flake8-bugbear/issues/445

This is considered a good practice to not use the same object that is being iterated (especially mutable, such as a class). Range make it clear that we iterate over N-times without direct use of `ireqs`. In simple use-case this should not matter but in more sophisticated ones it can be a little bit problematic. I think here, it's a matter of "explicit is better than implicit".

If you want to revert it later and cover with noqa/comments, feel free to do so.

Reply:
As example, let's imagine that `_` is replaced with `req`:
```python
    for req in ireqs:
        ireqs.start_async()
        results = req.outputs()
        req.infer(...)
    ireqs.wait_all()
```
It's clear what can go wrong here and there. So it's the matter of good practice but the rule can be omitted if developer is sure of the implementation. So that's why I have no strong opinion on which approach should we pick: "reverting-commenting" or leaving it as in this PR. I am just unblocking CIs for now.

Reply:
I wasn't going to block. Just curious.